packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -636,45 +636,43 @@ export function getNotSeenValidatorsFn(state: CachedBeaconStateAllForks): GetNot
   }
 
   // altair and future forks
-  else {
```

Comment:
do we really think this is a good change? Especially for larger code block being explicit with the `else` seems good to me as it does not rely on `if`-block to return or throw

Reply:
Was suggested by @wemeetagain snd I am also in favor of not using such else bocks. More levels of indentations make code less readable.

Reply:
> More levels of indentations make code less readable.

I don't think this can be applied generally, some cases I think identation can help make it more readable

Reply:
I would just go with the style that the original author of the code preferred, `else` is more explicit and can support readability imo.

Doesn't apply everything e.g. sometimes you just wanna throw an error and not use else, if it's a 1-liner

Reply:
If indentation is needed for readability, one can use code blocks. There is a reason community named such style `noUselessElse`.

Reply:
I tend to agree with that... else is a code smell of overly long code blocks that could be made more readable...